File: reimi/frontend/reimi_app/lib/domain/value_objects/weather_personality_polarity.dart
import 'package:json_annotation/json_annotation.dart';

enum WeatherPersonalityPolarity {
  @JsonValue('SENSITIVE')
  sensitive,

  @JsonValue('NEUTRAL')
  neutral,

  @JsonValue('PLANNED')
  planned,

  @JsonValue('FLEXIBLE')
  flexible,

  @JsonValue('OUTDOOR')
  outdoor,

  @JsonValue('INDOOR')
  indoor,

  @JsonValue('EMOTIONAL')
  emotional,

  @JsonValue('RATIONAL')
  rational,
}

File: reimi/backend/reimi_app/src/main/java/com/reimi/reimi_app/domain/model/weatherpersonality/type/WeatherPersonalityPolarity.java
package com.reimi.reimi_app.domain.model.weatherpersonality.type;

public enum WeatherPersonalityPolarity {
    SENSITIVE("S（高）"),
    NEUTRAL("N（低）"),
    PLANNED("P（計画型）"),
    FLEXIBLE("F（柔軟型）"),
    OUTDOOR("O（アウトドア）"),
    INDOOR("I（インドア）"),
    EMOTIONAL("E（情緒）"),
    RATIONAL("R（実用）");

    private final String label;

    WeatherPersonalityPolarity(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }
}